### PR TITLE
Package highlexer.0.1

### PR DESCRIPTION
**Pull request by EmileTrotignon**
### `highlexer.0.1`
An OCaml syntax highlighter based on the OCaml lexer

---
* Homepage: https://github.com/EmileTrotignon/highlexer
* Source repo: git+https://github.com/EmileTrotignon/highlexer.git
* Bug tracker: https://github.com/EmileTrotignon/highlexer/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by avsm** [comment]
This is failing across the board with

```
# Error: I don't know about package highlexer (passed through --only-packages)
```

**Comment by EmileTrotignon** [comment]
> This is failing across the board with
> 
> ```
> # Error: I don't know about package highlexer (passed through --only-packages)
> ```

I have seen this error, but I have no idea what it means.

**Comment by mseri** [comment]
I am rerunning the CI. It builds fine locally for me

File: packages/highlexer/highlexer.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/EmileTrotignon/highlexer.git"
```
Comment (mseri) [inline packages/highlexer/highlexer.0.1/opam:33]:
```suggestion
dev-repo: "git+https://github.com/EmileTrotignon/highlexer.git"
url {
  src: "https://github.com/EmileTrotignon/highlexer/archive/refs/tags/0.1.tar.gz"
  checksum: [
    "md5=5e6b2b4fa38383e0b3bc3a96be929a66"
    "sha256=9146a570641406dd8325682f57ce6062ad770f2dbe1fe5180ff607171e69bee2"
  ]
}
```
Thanks to @kit-ty-kate we found out the mystery! There is no archive here!

Comment (EmileTrotignon) [reply packages/highlexer/highlexer.0.1/opam:33]: Thanks a lot ! That's indeed an issue.

**Comment by mseri** [comment]
Please consider announcing this to discuss.ocaml.org. We have a community category and an announcement tag for this

**Pull request merged**